app/test/helpers/mocks.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:async';
+
+import 'package:acter/common/providers/notifiers/chat_notifiers.dart';
+import 'package:acter/features/home/providers/client_providers.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+
+import 'package:mockito/mockito.dart';
+
+/// Mocked version of ActerSdk
+class MockActerSdk extends Mock implements ActerSdk {}
+
+/// Mocked version of Acter Client
+class MockClient extends Mock implements Client {
+  @override
+  Stream<bool> subscribeStream(String topic) {
+    return Stream.value(true); // Return a dummy stream
+  }
+
+  @override
+  Future<Convo> convoWithRetry(String roomId, [int attempt = 0]) async {
+    return MockConvo();
+  }
+}
+
+/// Mocked version of Convo
+class MockConvo extends Mock implements Convo {
+  @override
```

Comment:
how about providing a value to expect here to ensure the _right_ thing is going to be sent...?